### PR TITLE
fix: SingleUser mode check function

### PR DESCRIPTION
**Pull request by fellipeh**
Change `run_cmd_in_pty` to `run_subprocess` into check single user function.

### CODE REVIEW

File: pyproject.toml [hunk 1]
```diff
@@ -0,0 +1,96 @@
+[tool.black]
```
Comment (SpyTec) [inline pyproject.toml:1]: File shouldn't be part of this PR

Comment (fellipeh) [reply pyproject.toml]: You're right... forgot to put into gitignore

File: convert2rhel/utils.py [hunk 2]
```diff
@@ -538,8 +538,9 @@ def require_single_user_mode():
     """
     loggerinst = logging.getLogger(__name__)
 
-    output, ret_code = run_cmd_in_pty("runlevel", print_cmd=False)
-    runlevel = output.replace("\r\n", "")[-1]
+    runlevel_path = pexpect.which("runlevel")
+    output, ret_code = run_subprocess(cmd=runlevel_path, print_cmd=False)
```
Comment (bocekm) [inline convert2rhel/utils.py:542]:
```suggestion
    output, ret_code = run_subprocess(cmd=runlevel_path, print_cmd=True, print_output=False)
```
See https://github.com/oamg/convert2rhel/pull/148.

File: convert2rhel/utils.py [hunk 3]
```diff
@@ -538,8 +538,9 @@ def require_single_user_mode():
...
+    runlevel = output.replace("\r\n", "").replace("\n", "")[-1]
```
Comment (bocekm) [inline convert2rhel/utils.py:543]:
```suggestion
    runlevel = output.split()[-1]
```
The output is `N <number>`, e.g. `N 1`. You can use `split()` to split it to `['N', '1']`.

**Review by bocekm — CHANGES_REQUESTED** [review]

File: convert2rhel/utils.py (hunk 3 above)
Comment (ZhukovGreen) [reply convert2rhel/utils.py:543]: maybe also int(....) to have a valid int

**Review by bocekm — APPROVED** [review]
